### PR TITLE
Adds Scope to Task, Comment, BOS, BOM and adds JobID to Job/ Updates …

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
Adds Scope to Task, Comment, BOS, BOM and adds JobID to Job/ Updates Whats new

### CODE REVIEW

**Review by jrippingale — APPROVED** [review]
Looks like what I was expecting.  Thank you!

**Comment by cwhanse** [comment]
I can't recall - are we changing this PR to use context-specific objects that inherit from Scope, or are we using Scope directly?

**Comment by kevinkirkl3y** [comment]
@cwhanse I merged main before making these changes. Created Object specific Scope objects and updated the WhatsNew.

**Review by jrippingale — APPROVED** [review]
Great!  Looks perfect!

File: docs/whatsnew/2207.1.0.rst [hunk 1]
```diff
@@ -1,11 +1,22 @@
 
 .. _whatsnew_220710:
 
-2207.1.0
+2207.1.1
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:4]:
```suggestion
2207.1.0
```

File: docs/whatsnew/2207.1.0.rst [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
 --------
 
 Element changes
 ~~~~~~~~~~~~~~~
+<<<<<<< HEAD
+* Adds JobID Element (:pull:`84`)
+* Adds DelegatesToAHJID, ExceptionDescription, AHJLevelCode (:pull:`91`)
+Object changes
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:12]:
```suggestion

Object changes
```

File: docs/whatsnew/2207.1.0.rst [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+~~~~~~~~~~~~~~
+* Add Scope, Measurement, EnergyACMeas, IrradGlobalHorizMeas, PowerACMeas, TempAmbMeas (:pull:`82`)
+* Removes ForecastScope (:pull:`82`)
+* Adds BOSScope to BillOfServices, BOMScope to BillOfMaterials, TaskScope to Tasks, CommentScope to Comments (:pull:`84`)
+Unit changes
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:17]:
```suggestion

Unit changes
```

File: docs/whatsnew/2207.1.0.rst [hunk 4]
```diff
@@ -17,6 +28,7 @@ Unit changes
 ~~~~~~~~~~~~
  * Add ScopeTypeItemType (:pull:`82`)
  * Adds AHJLevelCodeItemType and Enums (:pull:`91`)
+>>>>>>> b3df71b0b8871cda7f66c8e2bfb0b6a2f680b1c3
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:31]:
```suggestion
```

**Review by cwhanse — APPROVED** [review]
Some cleanup in the whatsnew. You can commit the suggestions directly on github.

File: docs/whatsnew/2207.1.0.rst [hunk 5]
```diff
@@ -6,6 +6,17 @@
...
+
+Object changes
+~~~~~~~~~~~~~~
+* Add Scope, Measurement, EnergyACMeas, IrradGlobalHorizMeas, PowerACMeas, TempAmbMeas (:pull:`82`)
+* Removes ForecastScope (:pull:`82`)
+* Adds BOSScope to BillOfServices, BOMScope to BillOfMaterials, TaskScope to Tasks, CommentScope to Comments (:pull:`84`)
+
+Unit changes
+~~~~~~~~~~~~
 *Adds DelegatesToAHJID, ExceptionDescription, AHJLevelCode (:pull:`91`)
```
Comment (cwhanse) [inline docs/whatsnew/2207.1.0.rst:20]:
```suggestion
* Adds DelegatesToAHJID, ExceptionDescription, AHJLevelCode (:pull:`91`)
```

**Pull request merged**